FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/FhirToDatalakeJobVersionManager.cs

```diff
@@ -83,5 +84,25 @@ public static class FhirToDatalakeJobVersionManager
             nameof(FhirToDataLakeProcessingJobInputData.DataStartTime),
             nameof(FhirToDataLakeProcessingJobInputData.ToBeProcessedPatients),
         };
+
+        // job version V4
```

Comment:
nit: add comments what is the difference comparing with job version V3.

Reply:
Added here, thanks.